docs/guide/units_tutorial.rst

```diff
@@ -0,0 +1,232 @@
+.. _units_tutorial:
+
+UNITS Tutorial 
+==============
+
+Units in models are checked for consistency with
+
+.. code::
+    python
+
+    modlunit file
+
+where *file* is the prefix to a model description file (having a ``.mod`` suffix).
+ 
+ Units are understood at the level of the unix units command. The syntax for units is any string understood by the unix units command and enclosed in parentheses. In particular, scientific notation is expressed as 1.111-5 or 1.111+5 instead of the more usual 1.111e-4, etc. Also an integer following a name is the power of that dimension, e.g. cm4 is centimeter to the fourth power. Lastly, all units in the numerator must come first followed by a slash character followed by all the units in the denominator. e.g a volt can be expressed either as (volt), (m2-kg/sec2-coul), or as (m m kg/sec sec coul). ie. one cannot have two slashes in a units expression.
+
+Consider the model description language fragment:
+
+.. code::
+    python
```

Comment:
this is NMODL not Python... but it doesn't know about noodle, so just omit language